meetings/2019-01-07.md

```diff
@@ -0,0 +1,115 @@
+
+* Standards engagement
+  - Refs:
+    - https://github.com/nodejs/bootstrap/pull/64
+    - https://github.com/nodejs/bootstrap/pull/63
+
+  - LittleDan: Outreach calls to get spec authors in touch with developers who will
+     use their outputs.
+  - LittleDan: Invite participants to occasionally attend standards meetings
+  - Myles: Interested in exploring partnerships between $Foundation and ECMA itself.
+  - Myles: Travel Fund like use for attendance of TC39 as “invited experts”. Process
+    for designating / inviting “invited experts” to help $Foundation processes here?
+
+### Discussion (40 minutes)
+
+* Organize the next collaborator summit for $foundation, not just Node.js  [#58](https://github.com/nodejs/bootstrap/issues/58)
+  - 10 minute timebox
+
+  - Myles: Recap. Have $Foundation budget for collab summit right before JS Conf EU.
+    TC39 also has a meeting right after. There will be many JSF members in attendance
+    as well. Do we want to make this a joint meetup?
+  - Jory: Idea. Open source “showcase” to show off all projects that will be in the foundation,
+     or those that may be interested. 
+  - Jory: Idea. At least one project interested in hosting a workshop as a part of this as well.
+  - Jory: These projects likely won’t be bringing in large groups for working group meetings, but
+     collaborator meetups and networking is helpful for the JSF community right now.
+  - Myles: For this first one the “Open Source Bazaar” structure may be beneficial.
+  - Dawson: We need to make additional space for this – third track so its an addition vs a 
+    subtraction  from dedicated Node.js. Also received feedback that single track content
+    is not as helpful as breakout sessions.
+  - Tierney: Let’s be cognisant of the time dedicated to organization of these events.
+  - Myles: Lots of meetups from “What the Fest” going on. Collaborate w/ JSConfEU to host a   
+    public meetup event? Depends on goals of JSF projects with Bazaar.
+  - 
+
+* Drafting initial governance for merged foundation [#33](https://github.com/nodejs/bootstrap/issues/33)
+  - 30 minute timebox
+  - Refs:
+    - https://docs.google.com/presentation/d/1qUcvZz4wmQtwcWu9rWjxFNmWw5plD9-4_7mtvQvCegk/
+    - What does C3 look like? [#38](https://github.com/nodejs/bootstrap/issues/38)
+    - What does the CPC look like? [#37](https://github.com/nodejs/bootstrap/issues/37)
+    - Top Level Council(s) Shape and Focus [#52](https://github.com/nodejs/bootstrap/issues/52)
+    - https://github.com/nodejs/bootstrap/projects/1
+    - [CPC Charter Proposal](https://github.com/nodejs/bootstrap/blob/master/proposals/mdawson-cpc-charter/README.md)
+
+  - Myles: Recap work done to differentiate CPC and C3 responsibilities in project board. Not
+    clear we need a hard seperation between CPC and C3. This is important for us to figure out 
+    sooner rather than later because it will affect allocation of board seats and by-laws.
+  - c 
```

Comment:
- Myles asked if there were any objections to moving forward with one group which would be the CPC.  
   No objections from those present, but will ask again in next meeting as not everybody can make each
   of the time slots.
- Myles: Does Michael's CPC charter cover board representation? If we just have the CPC then we need to
  factor that in.
- Michael: Not, yet. Will take action to create PR to add that in. (PR is now here: 
   https://github.com/nodejs/bootstrap/pull/70)